### ISSUE TITLE
[BUG] - When creating a new project, the database isn't being updated correctly

### ISSUE THREAD

**Issue by arungalva** [arungalva/4th-year-Project-Website#46]
**Describe the bug**
When I create a new project, it adds a new row to the projects table with the columns id, name, description, min students, max students filled correctly. However, the other columns that should be filled automatically aren't being filled. Also, the enum column has a weird value

**To Reproduce**
1. Go to /professor
2. Click on 'Add project'
3. Fill the form and hit submit
4. Query the database to see the bug

**Expected behavior**
The columns creation date, lastmodifieddate and project restriction should have correct values

**Actual behavior**
The columns creation date, lastmodifieddate are null and project restriction has the value 
```
¬ísr$java.util.EnumSet$SerializationProxyÓÛvTÊÑLelementTypetLjava/lang/Class;[elementst[Ljava/lang/Enum;xpvr(com.eggplant.emoji.entities.User$Programxrjava.lang.Enumxpur[Ljava.lang.Enum;¨ê-3Ò/xp~q~tBIOMEDICAL_ELECTRICAL~q~t
COMMUNICATION~q~tCOMPUTER_SYSTEMS~q~tSOFTWARE
```

**Screenshots**
![image](https://user-images.githubusercontent.com/12630802/53864337-3a887880-3fba-11e9-8a5a-a9bffe8d0b06.png)

**Additional context**

**Comment by arungalva** [comment]
This also points out an issue with our process where we need to start making sure that we have a lot more unit tests before anything gets merged. We should discuss this and work on it in from now

**Comment by ShasRanasinghe** [comment]
This issue has been fixed when master was changed to postgres in #56
